### PR TITLE
Add implementation for Signature created from mrenclave hash

### PR DESCRIPTION
**Pull request by lkatalin**
Refers to #29

### CODE REVIEW

**Comment by npmccallum** [comment]
@lkatalin I'm not sure what the value of a constructor from a hash without the other information is.

File: sgx-types/src/sig.rs [hunk 1]
```diff
@@ -110,6 +110,20 @@ pub struct Signature {
     pub q2: RsaNumber, // value used in RSA signature verification
 }
 
+impl Signature {
+    /// Creates a signature from a mrenclave hash
+    pub fn from_hash(hash: [u8; 32]) -> Self {
```
Comment (steveeJ) [inline sgx-types/src/sig.rs]: would it make sense to introduce a stronger type for the hash?

**Comment by npmccallum** [comment]
@lkatalin What's the status of this PR?

**Comment by lkatalin** [comment]
Closed because we are doing this from the `sgx-hasher` crate instead.

**Pull request closed without merging**